### PR TITLE
chore: update headings letter spacing

### LINKED ISSUE #48: [DESGIN BUG] Letter spacing on headings
all the headings should have 

letter-spacing: 0.04em;
Screen Shot 2022-11-25 03 57 06 PM

### PR DESCRIPTION
**Pull request by nhsz**
Fixes #48

### CODE REVIEW

**Review by corwintines — APPROVED** [review]

File: src/theme/foundations/textStyles.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ export const textStyles = {
     fontWeight: 700,
     fontSize: '2.75rem',
     lineHeight: '3.375rem',
-    letterSpacing: '0.05em',
+    letterSpacing: '0.04em',
```
Comment (wackerow) [inline src/theme/foundations/textStyles.ts:7]: I don't see where we're using 0.04 for an `h1`
image

Looks like we use 0.05 for the homepage h1's, and 0.03 for the other h1s

File: src/theme/foundations/textStyles.ts [hunk 2]
```diff
@@ -47,20 +47,21 @@ export const textStyles = {
...
-    letterSpacing: '0.02em',
+    letterSpacing: '0.04em',
     lineHeight: 'normal',
     color: 'body'
   },
   header6: {
     fontFamily: 'heading',
     fontSize: '0.875rem',
-    letterSpacing: '0.02em',
+    letterSpacing: '0.04em',
```
Comment (wackerow) [inline src/theme/foundations/textStyles.ts:57]: Same here... I'm seeing 0.02 on the design system 
image

@nloureiro could you maybe help clarify here?

Comment (nloureiro) [reply src/theme/foundations/textStyles.ts:57]: [desktop] h1 h2 h3 h4 >>>0.04rem  
[Mobile] h1 h2 h3 h4 >>>0.03rem  

[desktop] + [mobile] h5 h6 >>>> 0.02rem

does it help?

Comment (nhsz) [reply src/theme/foundations/textStyles.ts]: Fixed

**Review by corwintines — APPROVED** [review]

**Review by wackerow — APPROVED** [review]

**Pull request merged**